cache.rst

```diff
@@ -269,6 +269,44 @@ keys with a namespace that's generated by hashing the name of the pool, the name
 of the compiled container class and a :ref:`configurable seed<reference-cache-prefix-seed>`
 that defaults to the project directory.
 
+If you, for some reason, want to take control over pool namespace generation,
```

Comment:
What about this, to better explain when this is useful?:
`If you need the namespace to be interoperable with a third-party app, you can take control over auto-generation by setting the [...]`

Reply:
thanks, thats much better, indeed :)